### PR TITLE
fix(foundryup): warn macos users without libusb

### PR DESCRIPTION
**Pull request by transmissions11**

### CODE REVIEW

**Comment by transmissions11** [comment]
should close https://github.com/gakonst/foundry/issues/498

File: foundryup/install [hunk 1]
```diff
@@ -57,6 +57,11 @@ if [[ "$FOUNDRYUP_REPO" == "gakonst/foundry" && -z "$FOUNDRYUP_BRANCH" ]]; then
     ARCHITECTURE="amd64" # Amd.
   fi
   
+  # Warn MacOS users that they may need to manually install libusb via Homebrew:
+  if [[ "$PLATFORM" == "darwin" && ! -f /usr/local/opt/libusb/lib/libusb-1.0.0.dylib ]]; then
+    echo "warning: libusb not found. You may need to install it manually on MacOS via Homebrew.
```
Comment (gakonst) [inline foundryup/install:62]:
```suggestion
    echo "warning: libusb not found. You may need to install it manually on MacOS via Homebrew (`brew install libusb`)
```

**Review by gakonst — APPROVED** [review]

File: foundryup/install [hunk 2]
```diff
@@ -57,6 +57,11 @@ if [[ "$FOUNDRYUP_REPO" == "gakonst/foundry" && -z "$FOUNDRYUP_BRANCH" ]]; then
     ARCHITECTURE="amd64" # Amd.
   fi
   
+  # Warn MacOS users that they may need to manually install libusb via Homebrew:
+  if [[ "$PLATFORM" == "darwin" && ! -f /usr/local/opt/libusb/lib/libusb-1.0.0.dylib ]]; then
+    echo "warning: libusb not found. You may need to install it manually on MacOS via Homebrew (`brew install libusb`).
+  fi
+
```
Comment (appleseed-iii) [inline foundryup/install:64]: fyi, this warning doesn't pop up on foundry install, maybe because `echo $PLATFORM` returns nothing

Comment (appleseed-iii) [reply foundryup/install:64]: `$OSTYPE` could work though

```zsh
% echo $OSTYPE
darwin20.0
```

Comment (appleseed-iii) [reply foundryup/install:64]: oh just realizing you removed that warning entirely also. https://github.com/gakonst/foundry/pull/537

Comment (onbjerg) [reply foundryup/install:64]: feel free :smile:

**Pull request merged**